invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/button.overrides

```diff
@@ -55,10 +55,19 @@
   padding: 0;
 }
 
-.ui.button.fluid-mobile {
-  @media all and (max-width: @largestMobileScreen) {
-    width: 100%;
-    min-width: 0em;
-    display: block;
+.ui.button{
+  &.fluid-tablet {
+    @media screen and (min-width: @tabletBreakpoint) and (max-width: @largestTabletScreen) {
+      width: 100%;
+      min-width: 0em;
+      display: block;
+    }
+  }
+  &.fluid-mobile {
+    @media all and (max-width: @largestMobileScreen) {
+      width: 100%;
+      min-width: 0em;
+      display: block;
```

Comment:
it looks like fluid-tablet class is not needed ? since it is code duplication

I would replace it with 
```
&.fluid-responsive {
  @media screen and max-width: @largestTabletScreen {
 ```
 and there is no need to duplicate custom classes